src/components/AddItem.js

```diff
@@ -54,60 +64,52 @@ const AddItem = ({ token }) => {
   };
 
   return (
-    <div>
-      <h1>Add Item</h1>
-      <form onSubmit={handleSubmit}>
-        <label>
-          Item Name
-          <input
-            type="text"
-            placeholder="add your item here"
-            name="itemName"
-            required
-            value={formData.itemName}
+    <div className="container">
+      <form className="form-container" onSubmit={handleSubmit}>
+        <TextField
+          id="outlined-basic"
+          name="itemName"
+          required
+          value={formData.itemName}
+          onChange={handleFormChange}
+          label="Add New Item"
+          variant="outlined"
+        />
+        {error && <Error errorMessage="That item is already in your list" />}
+        <FormControl component="fieldset">
+          <FormLabel component="legend" hidden>
+            Time Frame
+          </FormLabel>
+          <RadioGroup
+            aria-label="time frame"
+            name="time frame"
             onChange={handleFormChange}
-          />
-          {error && <Error errorMessage="That item is already in your list" />}
-        </label>
-
-        <fieldset className="fieldset">
-          <legend>Time Frame</legend>
-          <label htmlFor="timeFrame"> How soon will you buy this again?</label>
-          <br />
-          <input
-            type="radio"
-            id="soon"
-            name="timeFrame"
-            defaultChecked
-            value="7"
-            onChange={handleFormChange}
-          />
-          <label htmlFor="soon"> Soon</label>
-          <br />
-          <input
-            type="radio"
-            id="kinda-soon"
-            name="timeFrame"
-            value="14"
-            onChange={handleFormChange}
-          />
-          <label htmlFor="kinda-soon"> Kinda Soon</label>
-          <br />
-          <input
-            type="radio"
-            id="not-soon"
-            name="timeFrame"
-            value="30"
-            onChange={handleFormChange}
-          />
-          <label htmlFor="not-soon">Not Soon</label>
-          <br />
-        </fieldset>
-        <button type="submit">
-          <span>
-            Add Item<i class="fas fa-cart-plus"></i>
-          </span>
-        </button>
+          >
+            <FormControlLabel
+              value="7"
+              control={<Radio id="soon" />}
+              label="Soon"
+            />
+            <FormControlLabel
+              value="14"
+              control={<Radio id="kinda-soon" />}
```

Comment:
The radios look awesome and I love the slight animation that they come with :-)

Reply:
Thanks Mike. I thought that list of imports was getting out of control!